Classes/Manager/Submanagers/OCTSubmanagerAvatars.m

```diff
@@ -7,6 +7,10 @@
 //
 
 #import "OCTSubmanagerAvatars.h"
+#import "OCTFileStorageProtocol.h"
+#import "OCTTox.h"
+
+static NSString *const userAvatarFileName = @"user_avatar";
```

Comment:
It would be better to use `kUserAvatarFileName` for constants to have save style all over the project.